bulk_data_gen/metaqueries/generate_data.go

```diff
@@ -18,6 +18,8 @@ var (
 	measKey = []byte("example_measurement")
 )
 
+const TagSig = "Tag-%d"
```

Comment:
I had originally made changes to this file because I was going to add a query type that would require you to know the specific values of tags, so using a `uuid` wouldn't work. I came up with this system of naming the tags like `Tag-23` where the number is psuedo-random which works just as well as random `uuid` values and allows you to predict what the tag values will actually be if your query needs them.

I didn't end up actually making this query type, but decided to keep this modification in place in case we want to benchmark queries like that in the future.

Reply:
Understood.